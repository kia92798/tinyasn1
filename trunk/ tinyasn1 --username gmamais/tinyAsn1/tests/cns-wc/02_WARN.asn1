MY-MODULE DEFINITIONS AUTOMATIC TAGS::= BEGIN


	MYSQOF ::= SEQUENCE SIZE(3) OF INTEGER

	MYSQOF2 ::= MYSQOF( WITH COMPONENT (1..10))

	myArray MYSQOF2 ::= { 1 ,2 ,30 }

	
END
