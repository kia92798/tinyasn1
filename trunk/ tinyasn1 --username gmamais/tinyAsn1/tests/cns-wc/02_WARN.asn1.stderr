Warning: value 'myArray'defined in line 8 does not conform to its type constraints
